staging/cert-manager-setup/templates/post-install-hook-job.yaml

```diff
@@ -13,31 +14,10 @@ spec:
     metadata:
       name: "wait-for-cert-manager-webhook"
     spec:
+      serviceAccountName: default
       restartPolicy: Never
       containers:
       - name: {{ .Chart.Name }}
         image: bitnami/kubectl:latest
         imagePullPolicy: IfNotPresent
-        command: ["kubectl", "wait", "--for=condition=Available", "--timeout=300s", "apiservice", "v1beta1.webhook.certmanager.k8s.io"]
----
-apiVersion: batch/v1
-kind: Job
-metadata:
-  name: {{ include "cert-manager-setup.fullname" . }}-sleep
-  labels:
-{{ include "cert-manager-setup.labels" . | indent 4 }}
-  annotations:
-    "helm.sh/hook": post-install,post-upgrade
-    "helm.sh/hook-weight": "-5"
-    "helm.sh/hook-delete-policy": hook-succeeded,before-hook-creation
-spec:
-  template:
-    metadata:
-      name: "sleep"
-    spec:
-      restartPolicy: Never
-      containers:
-      - name: {{ .Chart.Name }}
-        image: ubuntu:xenial
-        imagePullPolicy: IfNotPresent
-        command: ["sleep", "30"]
+        command: ["kubectl", "wait", "--for=condition=Available", "--timeout=360s", "apiservice", "v1beta1.webhook.certmanager.k8s.io"]
```

Comment:
Just double-checking, 360's is arbitrary? Just checking why we increased this by one minute?

Reply:
There was a sleep before in this job, adding more time in case we "need" it